features/get_filter_output.feature

```diff
@@ -0,0 +1,200 @@
+        "instance_id": "94310d8d-72d6-492a-bc30-27584627inst1",
+        "type": "",
+        "downloads": {},
+        "dimensions": null,
+          "state": "published",
+          "events": null,
+          "published": true,
+          "dataset": {
+                  "id": "",
+                  "edition": "",
+                  "version": 0
+           },
+           "population_type": "",
+          "downloads": {
+            "xls": {
+              "href": "http://localhost:23600/downloads/datasets/cantabular-flexible-example/editions/2021/versions/1.xls",
+              "private": "http://minio:9000/private-bucket/datasets/cantabular-flexible-example-2021-1.xls",
+              "size": "6944",
+              "public": "https://csv-exported.s3.eu-west-1.amazonaws.com/full-datasets/cpih01-time-series-v1.csv-metadata.xls",
+              "skipped": true
+            },
+            "csv": {
+              "href": "http://localhost:23600/downloads/datasets/cantabular-flexible-example/editions/2021/versions/1.csv",
+              "private": "http://minio:9000/private-bucket/datasets/cantabular-flexible-example-2021-1.csv",
+              "public": "https://csv-exported.s3.eu-west-1.amazonaws.com/full-datasets/cpih01-time-series-v1.csv-metadata.csv",
+              "size": "277",
+              "skipped": true
+            },
+            "csvw": {
+              "href": "http://localhost:23600/downloads/datasets/cantabular-flexible-example/editions/2021/versions/1.csv-metadata.json",
+              "private": "http://minio:9000/private-bucket/datasets/cantabular-flexible-example-2021-1.csvw",
+              "public": "https://csv-exported.s3.eu-west-1.amazonaws.com/full-datasets/cpih01-time-series-v1.csv-metadata.csvw",
+              "size": "607",
+              "skipped": true
+            },
+            "txt": {
+              "href": "http://localhost:23600/downloads/datasets/cantabular-flexible-example/editions/2021/versions/1.txt",
+              "private": "http://minio:9000/private-bucket/datasets/cantabular-flexible-example-2021-1.txt",
+              "public": "https://csv-exported.s3.eu-west-1.amazonaws.com/full-datasets/cpih01-time-series-v1.csv-metadata.txt",
+              "size": "530",
+              "skipped": true
+            }
+          },
+          "links": {
+            "filter_blueprint": {
+              "href": "test-href",
+              "id": "test-id"
+            },
+            "self": {
+              "href": "test-self-href",
+              "id": "test-self-id"
+            },
+            "version": {
+              "href": "test",
+              "id": "test"
+            }
+          }
+        }
+    """
+    And the HTTP status code should be "200"
+
+  Scenario: Filter Output not found
+    When I GET "/filter-outputs/not-exist"
+
+    Then I should receive the following JSON response:
+    """
+    {
+      "errors": ["failed to get filter output: mongo: no documents in result"]
```

Comment:
I don't think we should be exposing the details of our database tech to the public.. Elsewhere for a 404 we just return 'filter not found' with the 404 status code. You can do this with the errorMessage field of the error in the handler or elsewhere